distribution/docker/druid.sh

```diff
@@ -148,5 +148,10 @@ then
     echo "$DRUID_LOG4J" >$COMMON_CONF_DIR/log4j2.xml
 fi
 
-mkdir -p var/tmp var/druid/segments var/druid/indexing-logs var/druid/task var/druid/hadoop-tmp var/druid/segment-cache
+DRUID_DIRS_TO_CREATE=${DRUID_DIRS_TO_CREATE-'var/tmp var/druid/segments var/druid/indexing-logs var/druid/task var/druid/hadoop-tmp var/druid/segment-cache'}
+if [ "X${DRUID_DIRS_TO_CREATE}" != "X" ]
```

Comment:
nit, why not use `if [ -n "$VAR" ]` like elsewhere in the script?

Reply:
mostly muscle memory I think, didn't pay enough attention to other parts of script while I made the change.

changed to be more idiomatic .